src/browserLookups/cookie.js

```diff
@@ -82,20 +82,21 @@ const cookie = {
 export default {
   name: 'cookie',
 
-  lookup(options) {
-    let found;
-
-    if (options.lookupCookie && typeof document !== 'undefined') {
-      const c = cookie.read(options.lookupCookie);
-      if (c) found = c;
+  // Deconstruct the options object and extract the lookupCookie property
+  lookup({ lookupCookie }) {
+    if (lookupCookie && typeof document !== 'undefined') {
+      return cookie.read(lookupCookie) || undefined;
     }
 
-    return found;
+    return undefined;
```

Comment:
This is not strictly needed but would cause errors in ESlint unless I change the rules.